include/arch/arm64/scripts/linker.ld

```diff
@@ -48,8 +48,13 @@
 #define RAM_SIZE (CONFIG_SRAM_SIZE * 1K)
 #define RAM_ADDR CONFIG_SRAM_BASE_ADDRESS
 
+#define BSS_ALIGN
```

Comment:
It may be cleaner to not define this here but instead
`
#ifndef BSS_ALIGN
#define BSS_ALIGN
#endif
`
at the end of this if-elif block instead of undef'ing BSS_ALIGN in the middle.

Also, there is a type of "varialbe" in the commit message.

Reply:
Good suggestion! I'll fix that